### PR TITLE
[SPARK-16445][MLlib][SparkR] Fix @return description for sparkR mlp summary() method

### PR DESCRIPTION
**Pull request by keypointt**
## What changes were proposed in this pull request?

Fix summary() method's `@return` description for spark.mlp
## How was this patch tested?

Ran tests locally on my laptop.

### CODE REVIEW

**Comment by keypointt** [comment]
hi @felixcheung this is just a minor patch to SPARK-16445, where the `@return` description is not accurate and fixed here. Do you mind have a look here? Thanks a lot :)

File: R/pkg/R/mllib.R [hunk 1]
```diff
@@ -732,7 +733,6 @@ setMethod("summary", signature(object = "MultilayerPerceptronClassificationModel
             labelCount <- callJMethod(jobj, "labelCount")
             layers <- unlist(callJMethod(jobj, "layers"))
             weights <- callJMethod(jobj, "weights")
-            weights <- matrix(weights, nrow = length(weights))
```
Comment (felixcheung) [inline R/pkg/R/mllib.R]: could we add a test for this output?

**Comment by felixcheung** [comment]
cool thanks, just one comment

**Comment by keypointt** [comment]
There is already a test `expect_equal(length(summary$weights), 64)`
https://github.com/apache/spark/blob/master/R/pkg/inst/tests/testthat/test_mllib.R#L371

is it enough or we should add more tests?

**Comment by felixcheung** [comment]
since the format of the output is changing it would be great to check more than just the length in test

**Comment by keypointt** [comment]
how about adding type check? `expect_equal(typeof(summary$weights), "list")`

**Comment by HyukjinKwon** [comment]
@felixcheung I think this is a irrelevant comment but is there any issue for enabling Windows test for now? It seems it should have ran a test for this PR.

**Comment by felixcheung** [comment]
@keypointt let's have a check for the first few values perhaps?
@HyukjinKwon It seems it would be good to start running R tests though I'm not sure actually - I'd defer to @shivaram

**Comment by shivaram** [comment]
@HyukjinKwon I thought it should have run the tests ? Any ideas why its not getting picked up ? Is there some other set of steps that we need to do ? (I'm now investigating how it works for Apache Thrift)

**Comment by shivaram** [comment]
I think we need to file an INFRA ticket like https://issues.apache.org/jira/browse/INFRA-11294 -- I'll file one now

**Comment by shivaram** [comment]
Created https://issues.apache.org/jira/browse/INFRA-12590

**Comment by HyukjinKwon** [comment]
I will check this out as far as I can and be back.

**Comment by HyukjinKwon** [comment]
@dongjoon-hyun Meanwhile, do you mind if I ask whether you have any idea on this maybe?

**Comment by dongjoon-hyun** [comment]
Up to my knowledge, INFRA ticket made by @shivaram is enough for now. For Apache REEF, we filed the following INFRA issue like that.

https://issues.apache.org/jira/browse/INFRA-11411

**Comment by felixcheung** [comment]
LGTM

**Comment by shivaram** [comment]
Thanks @keypointt - Merging into master

**Pull request closed without merging**